README.rst

```diff
@@ -36,10 +36,14 @@ also be
 `downloaded <https://readthedocs.org/projects/cobrapy/downloads/>`_.
 
 Please use the `Google
-Group <http://groups.google.com/group/cobra-pie>`_ for help.
+Group <http://groups.google.com/group/cobra-pie>`_ for help. When posting
+questions, please refer to these `StackOverflow
+guidelines <https://stackoverflow.com/help/how-to-ask>`_. You will make our
```

Comment:
'By writing a well formulated question, with sufficient detail, you are much more likely to quickly receive a good answer!'